e2e/tests-dfx/identity_command.bash

```diff
@@ -81,6 +80,11 @@ teardown() {
     assert_match "Identity already exists"
 }
 
+@test "identity new: --force re-creates an identity" {
+    assert_command dfx identity new --disable-encryption alice
+    assert_command dfx identity new --disable-encryption --force alice
+}
```

Comment:
Can we assert that the principal changed, so that the test verifies that the command created a new private key for the identity?

Reply:
Of course! I should have thought of that